networks/monero/src/merkle.rs

```diff
@@ -2,14 +2,13 @@ use std_shims::vec::Vec;
 
 use crate::primitives::keccak256;
 
-pub(crate) fn merkle_root(root: [u8; 32], leafs: &[[u8; 32]]) -> [u8; 32] {
+/// Calculates the merkel root of the given tree. Equivalent to `tree_hash` in monero-core.
```

Comment:
Merkle is a proper noun as it's a name. Also, typo.

monero-core? monero-project/monero? Should we cite a link to tree_hash here?